lib/managers/files.js

```diff
@@ -257,16 +257,26 @@ Files.prototype.getReadStream = function(fileID, options, callback) {
  *
  * @param {string} fileID - Box ID of the file being requested
  * @param {Object} [options] - Additional options for the request. Can be left null in most cases.
+ * @param {string} [options.extension] - The file format for the thumbnail. Default is png.
  * @param {Function} [callback] - Passed the thumbnail file or the URL to a placeholder thumbnail if successful.
  * @returns {Promise<Object>} A promise resolving to the thumbnail information
  */
 Files.prototype.getThumbnail = function(fileID, options, callback) {
+	var extension = 'png';
+
+	if (options && options.extension) {
+		if (options.extension === 'jpg') {
+			extension = options.extension;
+		}
+		delete options.extension;
```

Comment:
@PJSimon I used `delete` in order to keep consistency with the other methods, I would advise at least a shallow copy of the options and delete and pass to qs. If I were to use a loop to request jpg thumbnails and pass the same config object, after the first one I would get png as the extension property will be deleted. This is repeated in many places in the file.